yellow-paper/docs/contracts/da.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Data Availability (and Publication)
+---
+
+:::info
+This page is heavily based on the Rollup and Data Ramblings documents. As for that, I highly recommend reading [this very nice post](https://dba.xyz/do-rollups-inherit-security/) that Jon Charbonneau wrote.
+:::
+
+- **Data Availability**: The data is available to me right now
+- **Data Publication**: The data was available for a period when it was published.
```

Comment:
Is this the way that these terms are usually defined?

Reply:
Not really. Often people don't split them in two and just refer to it as Data Availability. Bartek from L2Beat have mentioned them as different a couple of times though: https://x.com/bkiepuszewski/status/1696417258552033558

Reply:
Maybe I'm mis-reading the tweet, it seems to imply that when we say DA we really mean data publishing, whereas I read this document to mean that DA and data publishing are different. Is that the case?

Reply:
When people say DA they are mostly talking about Data Publication. Where I am saying Data Availability is that the data is available beyond publication, e.g., what Bartek refers to as storage. Think he used entirely different words to not end up with "DA" is Data publication and Data Availability is different, where I would like to use the DA for data availability and then we can call data publication DP or something else.